pkg/login/flow_test.go

```diff
@@ -97,34 +97,49 @@ func TestTriggerMethodOIDCWithLoginServer(t *testing.T) {
 
 func TestTriggerMethodOIDCWithoutLoginServer(t *testing.T) {
 
-	// We're simulating a container environment where the browser can't be opened.
-	// See https://jira.mesosphere.com/browse/DCOS_OSS-5591
-	opener := open.OpenerFunc(func(_ string) error {
-		return errors.New("couldn't open browser, I'm just a container")
-	})
+	fixtures := []struct {
+		provider *Provider
+	}{
+		{
+			// We're simulating a container environment where the browser can't be opened.
+			// See https://jira.mesosphere.com/browse/DCOS_OSS-5591
+			defaultOIDCImplicitFlowProvider(),
+		},
+		{
+			// Non-regression test for a panic on browser login flows other than auth0.
+			// https://jira.mesosphere.com/browse/DCOS-60349
+			shibbolethLoginProvider(),
+		},
+	}
 
-	expectedLoginToken := "dummy_login_token"
-	expectedACSToken := "dummy_acs_token"
+	for _, fixture := range fixtures {
+		expectedLoginToken := "dummy_login_token"
+		expectedACSToken := "dummy_acs_token"
```

Comment:
These can probably be added to the the above `fixtures` / `tests` struct. Makes it easier to add additional tests and modify them in the future.